dev/create-release/release-build.sh

```diff
@@ -191,9 +191,17 @@ if [[ "$1" == "package" ]]; then
   make_binary_release() {
     NAME=$1
     FLAGS="$MVN_EXTRA_OPTS -B $BASE_RELEASE_PROFILES $2"
+    # BUILD_PACKAGE can be "withpip", "withr", or both as "withpip,withr"
     BUILD_PACKAGE=$3
     SCALA_VERSION=$4
 
+    if [[ $BUILD_PACKAGE == *"withpip"* ]]; then
```

Comment:
@vanzin what do you think of this approach? It simplifies the logic below too, avoiding repeating the main build step 3 times.

Reply:
Looks fine. Using wildcards is a little weird but I guess that's the cleanest way in bash.

But shouldn't you initialize PIP_FLAG and R_FLAG to empty before these checks?